### ISSUE TITLE
Show which Jira field was changed

### ISSUE THREAD

**Issue by Thihup** [Thihup/improved-jira-history-diff#5]
If the title changes, for instance, it will only show the diff, but will not show that the title field was changed.
On the other way around, it is possible to limit to just the description changes.

**Comment by dtruebin** [comment]
> it is possible to limit to just the description changes

Agreed, there is not much sense in showing rich diff for changes of status and simple fields. Description is where it's required most often, and, if possible, custom multiline text fields.
